src/custom/state/orders/reducer.ts

```diff
@@ -158,20 +158,24 @@ export default createReducer(initialState, (builder) =>
       const { order, id, chainId } = action.payload
 
       const orderStateList = order.status === OrderStatus.PRESIGNATURE_PENDING ? 'presignaturePending' : 'pending'
+      order.openSince = OrderStatus.PRESIGNATURE_PENDING ? undefined : Date.now()
 
       addOrderToState(state, chainId, id, orderStateList, order)
     })
     .addCase(preSignOrders, (state, action) => {
       prefillState(state, action)
       const { ids, chainId } = action.payload
 
+      const now = Date.now()
+
       ids.forEach((id) => {
         const orderObject = getOrderById(state, chainId, id)
 
         if (orderObject) {
           deleteOrderById(state, chainId, id)
 
           orderObject.order.status = OrderStatus.PENDING
+          orderObject.order.openSince = now
```

Comment:
Tracking when an order is open on order reducer, both when adding a pending order that's not presign and when orders are effectively presigned